docs/user-guide/api-mediation/configuration-saf-resource-checking.md

```diff
@@ -10,44 +10,47 @@ API ML can check for the authorization of the user on certain endpoints. Access
 
 Verification of the SAF resource is provided by the following three providers:
 
-- **`endpoint`**  
-This is the highest priority provider, such as a REST endpoint call (ZSS or similar one). This option is disabled by default. In Zowe, ZSS has the API to check for SAF   resource authorization.
-  
 - **`native`**  
-The Native JZOS classes from Java are used to determine SAF resource access. This is the default provider.
-  
+  The Native JZOS classes from Java are used to determine SAF resource access. This is the default provider.
+
+  **Note:** This provider cannot be used off-platform.
+
+- **`endpoint`**  
+The Endpoint provider relies on APIs such as a REST endpoint call (ZSS or similar one). This option is disabled by default. In Zowe, ZSS provides the API to check for SAF resource authorization.
```

Comment:
I wonder if it can be considered a breaking change (for anyone relying on the default being endpoint)

Reply:
Add some note or clarification that this is the default from 3.1

Reply:
Well in theory the default was still native, as long as they didn't set `endpoint.enabled` to true.